core/templates/dev/head/pages/community-dashboard-page/services/translate-text.service.ts

```diff
@@ -92,25 +93,10 @@ angular.module('oppia').factory('TranslateTextService', [
       },
       suggestTranslatedText: function(
           translationHtml, languageCode, successCallback) {
-        var url = '/suggestionhandler/';
-        var data = {
-          suggestion_type: 'translate_content',
-          target_type: 'exploration',
-          description: 'Adds translation',
-          target_id: activeExpId,
-          target_version_at_submission: activeExpVersion,
-          assigned_reviewer_id: null,
-          final_reviewer_id: null,
-          change: {
-            cmd: 'add_translation',
-            content_id: activeContentId,
-            state_name: activeStateName,
-            language_code: languageCode,
-            content_html: stateWiseContents[activeStateName][activeContentId],
-            translation_html: translationHtml
-          }
-        };
-        $http.post(url, data).then(successCallback);
+        TranslateTextBackendService.suggestTranslatedText(
+          translationHtml, languageCode, activeExpId, activeExpVersion,
+          activeContentId, activeStateName, stateWiseContentIds)
+          .then(successCallback());
```

Comment:
Have you tested this? (I think the param for `then` should be a function)

Reply:
Yes. My bad. Fixed this.

Reply:
(Make sure to test the behavior manually (twice).) Thanks for the fix! :)